proto/p4/config/v1/p4info.proto

```diff
@@ -274,17 +275,12 @@ message DirectCounter {
 }
 
 message MeterSpec {
-  // Corresponds to 'type' attribute for meter in P4 spec.
+  // Corresponds to 'type' constructor parameter for Meter / DirectMeter in PSA
   enum Unit {
     UNSPECIFIED = 0;
     BYTES = 1;
     PACKETS = 2;
   }
-  // Not part of the P4 spec yet but will be in the future.
-  enum Type {
-    COLOR_UNAWARE = 0;  // default value
-    COLOR_AWARE = 1;
-  }
   Unit unit = 1;
   Type type = 2;
```

Comment:
This pull request deletes the 'Type' enum, but the type field is still present and defined as 'Type'.

Reply:
Thanks for catching this Tom. It made me realize that the Travis CI build was not exiting properly in case of protoc failure so I was able to fix that. PTAL.